ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_QMLSamples/ArcGISRuntimeSDKQt_QMLSamples.pro

```diff
@@ -87,43 +87,43 @@ INCLUDEPATH += \
     $$COMMONVIEWER/SyntaxHighlighter
 
 HEADERS += \
```

Comment:
Because both the C++ and QML sampleviewer share a handful of source items that needed to be updated to accommodate the changes to bring zlib into the sampleviewer, i opted to just comment out the files that had shared classes and gave the qml sampleviewer a new mainSample.cpp that will build. It obviously doesn't work like it should but this is going away anyways and it avoids more scope creep in other areas.